tornado/log.py

```diff
@@ -54,18 +54,20 @@
 
 
 def _stderr_supports_color():
-    color = False
-    if hasattr(sys.stderr, 'isatty') and sys.stderr.isatty():
-        if curses:
-            try:
+    try:
+        if hasattr(sys.stderr, 'isatty') and sys.stderr.isatty():
+            if curses:
                 curses.setupterm()
                 if curses.tigetnum("colors") > 0:
-                    color = True
-            except Exception:
-                pass
-        elif colorama:
-            color = True
-    return color
+                    return True
+            elif colorama:
+                if sys.stderr is getattr(colorama, 'wrapped_stderr', object()):
```

Comment:
Hmm, doesn't look like it, but this is what you had in the previous PR. I'm trying to dust off a windows VM to test this. Looks like this will probably need to do a different check like you have in #2014. You've changed there from looking at colorama.initialize.wrapped_stderr to an isinstance check; do you have a particular reason to prefer one form over another? Neither appears to be documented.